# Order is stacked in reverse so that when viewing in terminal, interesting 
# data is outputted last (don't have to scroll up past an image and dvc!)
sensor_msgs/Image queryImage  # image grabbed by publisher
LabelDetails data
std_msgs/Header header
